consensus/parlia/parlia.go

```diff
@@ -391,7 +391,7 @@ func getVoteAttestationFromHeader(header *types.Header, chainConfig *params.Chai
 func getSignRecentlyLimit(blockNumber *big.Int, validatorsNumber int, chainConfig *params.ChainConfig) int {
 	limit := validatorsNumber/2 + 1
 	if chainConfig.IsBoneh(blockNumber) {
-		limit = validatorsNumber*2/3 + 1
+		limit = int(math.Ceil(float64(validatorsNumber) * 2 / 3))
```

Comment:
Please do not use math.Ceil which is for float, we can invent one for int64.

Reply:
fixed